compiler/rustc_typeck/src/check/upvar.rs

```diff
@@ -384,7 +384,7 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
     ///
     /// InferBorrowKind results in a structure like this:
     ///
-    /// ```
+    /// ```text
```

Comment:
This seems like it should be part of https://github.com/rust-lang/rust/pull/79816. Is there a reason it's also here?

Reply:
That may have been me mixing up two compilers while switching branches and not paying attention, I'll change it back